src/libs/actions/Report.js

```diff
@@ -1076,38 +1057,21 @@ function subscribeToNewActionEvent(reportID, callback) {
  * @param {String} reportID
  * @param {Object} action
  */
-function viewNewReportAction(reportID, action) {
-    const isFromCurrentUser = action.actorAccountID === currentUserAccountID;
-    const updatedReportObject = {};
-
-    // When handling an action from the current user we can assume that their last read actionID has been updated in the server,
-    // but not necessarily reflected locally so we will update the lastReadSequenceNumber to mark the report as read.
-    updatedReportObject.maxSequenceNumber = action.sequenceNumber;
-    if (isFromCurrentUser) {
-        updatedReportObject.lastReadTimestamp = Date.now();
-        updatedReportObject.lastReadSequenceNumber = action.sequenceNumber;
-    }
-
-    if (reportID === newActionSubscriber.reportID) {
-        newActionSubscriber.callback(isFromCurrentUser, updatedReportObject.maxSequenceNumber);
-    }
-
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT}${reportID}`, updatedReportObject);
-
-    const notificationPreference = lodashGet(allReports, [reportID, 'notificationPreference'], CONST.REPORT.NOTIFICATION_PREFERENCE.ALWAYS);
+function presentReportActionNotification(reportID, action) {
```

Comment:
NAB: Rename method to `showActionNotification` or `notifyAction`? We are already in `Report.js` so having Report in this method name is redundant IMO.

Reply:
I like "show" instead of "present" but I think keeping it as `reportAction` is clearer, since that's what the onyx collection/object is called.